aict-MERN/backend-of-aict-site-w-react/routes/posts.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express'
+import {getPosts, postPost} from '../db.js'
+const router = express.Router()
+
+router.route('/posts').get(async (req,res)=> {
+    try {
+        let posts = await getPosts()
+        res.status(200).json({posts})
+    } catch (error) {
+        res.status(404).json({msg: 'FAILED'})
+        console.log(error)
+    }
+}).post(async(req,res) => {
+    const values = [req.body.posted_by, req.body.post, new Date().toLocaleDateString('en-CA')]
+    try {
+        await postPost(values)
```

Comment:
its usually neat when such a function returns a boolean saying whether it has completed successfully or not